.github/workflows/continous-integration.yml

```diff
@@ -191,3 +195,8 @@ jobs:
         RASA_PRO_BETA_INTENTLESS: true
       run: |
         make test-failing | grep '0 passed'
+
+    - name: Stop Duckling server
```

Comment:
Same here

Reply:
Not sure i understand why do I need a make command for this instead of running directly the docker commands?

Reply:
Help to keep things consistent between CI and when someone wants to do same thing on localmachine.